### PR TITLE
cash_only-task_1

### PR DESCRIPTION
**Pull request by oscargtoro**

### CODE REVIEW

File: web_dynamic/0-hbnb.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+app = Flask(__name__)
+
+
+@app.teardown_appcontext
+def close_db(error):
+    """ Remove the current SQLAlchemy Session """
+    storage.close()
+
+
+@app.route('/hbnb', strict_slashes=False)
```
Comment (lh1008) [inline web_dynamic/0-hbnb.py]: Hi @oscargomez87,

I'm checking and the route had to be updated to `/0-hbnb/`.

**Review by lh1008 — COMMENTED** [review]
Hey @oscargomez87 

I see task 0 has everything, I just left you a comment about the route in file `0-hbnb.py`.

File: web_dynamic/0-hbnb.py (hunk 1 above)
Comment (oscargtoro) [reply web_dynamic/0-hbnb.py]: route updated

**Review by KevinCastroP — COMMENTED** [review]
Ok, the route was changing. All good now

**Review by KevinCastroP — APPROVED** [review]
all good now, the route was change

**Pull request merged**